Index: run.py
===================================================================
diff --git a/run.py b/run.py
--- a/run.py	
+++ b/run.py	
@@ -1,0 +1,28 @@
+import torch
+torch._dynamo.config.cache_size_limit = 64
+torch._dynamo.config.suppress_errors = True
+torch.set_float32_matmul_precision('high')
+import ChatTTS
+from wave import Wave_write
+import numpy as np
+chat = ChatTTS.Chat()
+chat.load_models(source='local', local_path='.\ChatTTS')
+
+text = "大家好，我是沐柚" #自定义文本
+
+#添加笑声，停顿等
+params_refine_text = {
+  'prompt': '[oral_2][laugh_0][break_4]'
+}
+chat.infer(text, refine_text_only=True)
+
+wavs = chat.infer(text, params_refine_text=params_refine_text)[0]
+
+sample_rate = 24000
+
+audio_data_rescaled = (wavs * 28000).astype(np.int16).flatten() #转换数据类型到合适的范围
+
+# 创建并打开一个wav文件用于写入，保存在./Audio目录下
+with Wave_write('./Audio/test_muyou01.wav') as wave_file:
+    wave_file.setparams((1, 2, sample_rate, len(audio_data_rescaled), 'NONE', 'not compressed'))
+    wave_file.writeframes(audio_data_rescaled.tobytes())
